src/Form/Type/GiftCardConfigurationType.php

```diff
@@ -43,4 +45,18 @@ public function getBlockPrefix(): string
     {
         return 'setono_sylius_gift_card_gift_card_configuration';
     }
+
+    public function configureOptions(OptionsResolver $resolver): void
+    {
+        parent::configureOptions($resolver);
+        $resolver->setDefault('validation_groups', function (FormInterface $form): array {
+            $validationGroups = $this->validationGroups;
+            $data = $form->getData();
+            if (null === $data->getId()) {
+                $validationGroups[] = 'setono_sylius_gift_card_create';
+            }
+
+            return $validationGroups;
+        });
+    }
```

Comment:
I don't understand why this is needed?

Reply:
Oh, it is not anymore. It was on my first attempt. I will remove that